README.md

```diff
@@ -0,0 +1,146 @@
+
+Google, Waymo, Red Cross, Omdena, and Rarebase use Hub.
+
+## Features 
+### Current Release
+
+* Easy dataset creation and hosting on Activeloop Cloud or S3
+* Rapid dataset streaming to any machine
+* Simple dataset integration to PyTorch with no boilerplate code
+* Dataset query using custom filter functions without having to download the entire dataset
+* Rapid data processing using transforms on distributed compute
+* Linear data pipelines
+* Rapid [visualization](http://app.activeloop.ai/?utm_source=github&utm_medium=repo&utm_campaign=readme) of image datasets via integration with Activeloop Platform
+ <p align="center">
+    <br>
+    <img src="https://raw.githubusercontent.com/activeloopai/Hub/master/docs/visualizer%20gif.gif" width="75%"/>
+    </br>
+Visualization of a dataset uploaded to Hub via <a href = "https://app.activeloop.ai/datasets/popular?utm_source=github&utm_medium=readme&utm_campaign=desc">app.activeloop.ai</a>.
+</p>
+
+### Coming Soon
+
+* Datasets hosting on Google Cloud and Azure
+* Datasets integration to TensorFlow
+* Dataset version control
+* Dataset query using text-based query language
+* Loading of data in random order without having to download the entire dataset
+* DAG and continuous pipelines
+
+## How does Hub work?
+
+Databases, data lakes, and data warehouses are best suited for tabular data and are not optimized for deep-learning applications using data such as images, videos, and text. Hub is a Data 2.0 solution that stores datasets as chunked compressed arrays, which significantly increases data transfer speeds between network-connected machines. This eliminates the need to download entire datasets before running code, because computations and data streaming can occur simultaneously without increasing the total runtime.
+
+Hub also significantly reduces the time to build machine learning workflows, because its API eliminates boilerplate code that is typically required for data wrangling ✌️.
+
+## Getting Started with Hub
+Hub is written in 100% python and can be quickly installed using pip.
+```sh
+pip3 install hub
+```
+Accessing datasets in Hub requires a single line of code. Run this snippet to get the first thousand images in the [MNIST database](https://app.activeloop.ai/dataset/activeloop/mnist/?utm_source=github&utm_medium=repo&utm_campaign=readme) in the numpy array format:
+```python
+from hub import Dataset
+
+mnist = Dataset("activeloop/mnist")
+mnist_np = mnist["image"][0:1000].compute()
+```
+To access and train a classifier on your own Hub dataset stored in cloud, run:
+```python
+my_dataset = Dataset("http://s3.amazonaws.com/[my_bucket_name]/")
+my_dataset_pytorch = my_dataset.to_pytorch(lambda x: (x["image"], x["label"]))
+
+train_loader = torch.utils.data.DataLoader(my_dataset_pytorch, batch_size=1, num_workers=0)
+
+## Training Loop Here ##
+```
+
+## Documentation
```

Comment:
This should be at the very top of the page

Reply:
often developers go to the github with the sole purpose of finding documentation. i shouldn't have to look for this too hard

Reply:
The positioning of all the elements is a product of optimizing against many constraints, so I'd prefer to not change it without going through all of the conversations that Vinn Mika and I had. I'm happy to go through our readme re-write philosophy in order to explain our thought process.

Our approach in the new readme is to focus less on code and more on highlighting Hub's awesome features. For checking out code, we're pointing users to Gitbook, where we will have an awesome mix of getting started code as well as in-depth applications.

Reply:
maybe can we add a link to docs (something tiny like [docs]()) at both the top and bottom of the page? like a header and footer?

Reply:
Yup, there's a link to the gitbook. We can add a separate link to the API reference, even though there's a link to the API reference in the gitbook.